src/test/python/FibSpec.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from src.main.python.Fibonacci import fib
+
+
```

Comment:
Thanks for observing and adhering to the style for this set of tests. Good work.